### PR TITLE
Help documentation

### PR DESCRIPTION
**Pull request by zmedelis**

### CODE REVIEW

File: front-end/packages/help/getting-started.md [hunk 1]
```diff
@@ -1,26 +1,114 @@
...
+*Data* sidebar provides the tools to load CSV data files to be used in generation. Note that CSV files have to be comma separated with the column headers at the top.
 
-Vestibulum vitae odio vel odio gravida tincidunt. Phasellus vehicula, ex sed vehicula semper, arcu turpis tincidunt lacus, quis egestas augue lectus non tellus. Maecenas est nunc, semper eget tellus nec, congue dictum libero. Nunc sit amet tellus maximus, mattis est eget, scelerisque felis. Mauris dapibus eros nec ipsum venenatis facilisis. Maecenas interdum auctor neque, accumsan dapibus erat. Vivamus vitae ullamcorper quam. Nulla blandit sapien id eleifend blandit. Curabitur id volutpat enim. Nunc tellus mauris, blandit nec fringilla vitae, sollicitudin in ipsum. Cras viverra sapien felis, et lobortis diam luctus at. Quisque et dui id turpis finibus accumsan sed eu dolor.
+More than one data file can be loaded with the help of *Add* button. File selected in the drop down menu is a current one and will be used for the generation.
```
Comment (rokasramas) [inline front-end/packages/help/getting-started.md]: "is a current" -> "is the current", "the generation" -> "generation" or "text generation"

File: front-end/packages/help/getting-started.md [hunk 2]
```diff
@@ -1,26 +1,114 @@
...
 
-Aliquam maximus augue a augue auctor tempor. Suspendisse sit amet dolor consectetur, bibendum orci sit amet, ornare elit. Etiam mauris diam, iaculis eu nunc non, mattis molestie nunc. Vivamus tempus volutpat augue at aliquet. Suspendisse potenti. Pellentesque vestibulum commodo tincidunt. Ut vel est suscipit ex dignissim imperdiet. Vivamus rhoncus eros metus, ut ornare arcu semper a. Morbi id pretium turpis. Sed sed nisi sed tortor placerat pulvinar. Quisque cursus felis ex, et vestibulum lacus sodales a. Nunc non finibus metus. Ut gravida, enim sed laoreet commodo, nisl turpis lobortis ante, iaculis dictum sapien nibh eget nibh. Phasellus accumsan tempor mi a gravida. Vivamus imperdiet dictum nibh, nec pulvinar mi rhoncus in. Maecenas molestie posuere blandit.
+Contents of the current data file are displayed in the table bellow. Active row can be selected with the controls at the top of the table.
```
Comment (rokasramas) [inline front-end/packages/help/getting-started.md]: "are displayed" -> "is displayed"

File: front-end/packages/help/getting-started.md [hunk 3]
```diff
@@ -1,26 +1,114 @@
...
+![data](help/language.png)
 
-Aliquam maximus augue a augue auctor tempor. Suspendisse sit amet dolor consectetur, bibendum orci sit amet, ornare elit. Etiam mauris diam, iaculis eu nunc non, mattis molestie nunc. Vivamus tempus volutpat augue at aliquet. Suspendisse potenti. Pellentesque vestibulum commodo tincidunt. Ut vel est suscipit ex dignissim imperdiet. Vivamus rhoncus eros metus, ut ornare arcu semper a. Morbi id pretium turpis. Sed sed nisi sed tortor placerat pulvinar. Quisque cursus felis ex, et vestibulum lacus sodales a. Nunc non finibus metus. Ut gravida, enim sed laoreet commodo, nisl turpis lobortis ante, iaculis dictum sapien nibh eget nibh. Phasellus accumsan tempor mi a gravida. Vivamus imperdiet dictum nibh, nec pulvinar mi rhoncus in. Maecenas molestie posuere blandit.
+### Text Generation Results
 
+This section will show the resulting texts. Each block of text represents variations of the text given the same document plan and current row of data. A flag next to text shows the language of the text.
```
Comment (rokasramas) [inline front-end/packages/help/getting-started.md]: "block of text" -> "block", "variations of the text" -> "variations of text", "the language of the text" -> "the language"

File: front-end/packages/help/getting-started.md [hunk 4]
```diff
@@ -1,26 +1,114 @@
...
+
+![results](help/results.png)
+
+In cases when no text can be generated (due to missconstructed document plan), you will see 'No Variants.' message.
```
Comment (rokasramas) [inline front-end/packages/help/getting-started.md]: In cases where no text was generated due to misconstructed document plan

File: front-end/packages/help/getting-started.md [hunk 5]
```diff
@@ -1,26 +1,114 @@
...
+
+### AMR
+
+This is a core element of the Accelerated Text engine. AMR - Abstract Meaning Representations - define what kind of messages can be constructed about the data. Choose from the AMR list bellow what you want to communicate to your users. AMR must be dragged to the Document Plan canvas to the left (Document Plan construction elements are described in the section bellow).
```
Comment (zaibacu) [inline front-end/packages/help/getting-started.md]: `to the Document Plan canvas to the left` a bit too much repetition of `to the`, maybe `to the Document Plan on the left`

Comment (rokasramas) [inline front-end/packages/help/getting-started.md]: "a core element" -> "the core element"; "AMR -" -> "AMRs -"; "AMR must" -> "AMR block must"

File: front-end/packages/help/getting-started.md [hunk 6]
```diff
@@ -1,26 +1,114 @@
...
+* *a.n.* - adjective-noun construction to specify expressions like 'red shoe' (where 'a.' is 'red' and 'n.' is 'shoe')
+
+![valuestb](help/values-tb.png)
+
+### Lists
+
+Handling of list elements - specify in which order elements enclosed in the list should be used.
+
+* *sequence* - exact sequence of the enclosed elements.
+* *in random order* - shuffle enclosed elements in a random order, thus producing variations in the text.
```
Comment (rokasramas) [inline front-end/packages/help/getting-started.md]: "variations in the text" -> "text variations"

File: front-end/packages/help/getting-started.md [hunk 7]
```diff
@@ -1,26 +1,114 @@
...
+* *a synonym from* - chose one of the elements in the list, thus producing variations in the text.
```
Comment (rokasramas) [inline front-end/packages/help/getting-started.md]: same as above

File: front-end/packages/help/getting-started.md [hunk 8]
```diff
@@ -1,26 +1,114 @@
...
+
+![liststb](help/lists-tb.png)
+
+### Logics
+
```
Comment (rokasramas) [inline front-end/packages/help/getting-started.md]: I think "Logic" is more appropriate

File: front-end/packages/help/getting-started.md [hunk 9]
```diff
@@ -1,26 +1,114 @@
...
+Text generation logic sometimes has to branch depending on the data supplied. Use *Logics* elements to control branching.
```
Comment (rokasramas) [inline front-end/packages/help/getting-started.md]: Logics -> Logic

File: front-end/packages/help/getting-started.md [hunk 10]
```diff
@@ -1,26 +1,114 @@
...
+
+* *if-then-else* - main construct to specify branching logics, if a condition satisfies a *check* (see next section), then perform an action, else do some other action.
```
Comment (rokasramas) [inline front-end/packages/help/getting-started.md]: logics again

File: front-end/packages/help/getting-started.md [hunk 11]
```diff
@@ -1,26 +1,114 @@
...
+
+![logicstb](help/logics-tb.png)
+
+### Checks
+
+Checks are used in combination with *Logics* elements. Note that condition in the check element can be chosen from the drop down list.
+
+* *=* - 'equality' condition
+* *>* - 'more than' condition
+* *is in* - this one allows to check if your value is in a list of certain values, for example is brand 'X' in a list of brands 'X, Y, Z'.
```
Comment (rokasramas) [inline front-end/packages/help/getting-started.md]: *is in* checks if a string contains substring

**Pull request merged**